blackbox/docs/sql/administration/ingestion_rules.txt

```diff
@@ -24,58 +24,55 @@
 Ingestion Rules
 ===============
 
-In CrateDB, ingestion rules define where and how the incoming data of a
+Ingestion rules define where and how the incoming data of a
 specific ingestion should be routed and stored.
 
-In this section, we will learn about creating and deleting ingestion rules,
-listing the existing ingestion rules and finally the required privileges for
-the ``CREATE`` and ``DROP INGEST RULE`` statements, when the
-:ref:`administration_user_management` feature is enabled.
+This document will show you how to:
 
-Creating and Deleting Ingestion Rules
-.....................................
+- Create ingestion rules
```

Comment:
should these be links?

Reply:
i think it's not needed, the document is quite small